src/main/java/org/zalando/zmon/notifications/NotificationServiceApplication.java

```diff
@@ -55,9 +56,17 @@
     EscalationConfigSource escalationConfigSource;
 
     @Bean
-    TokenInfoService getTokenInfoService(NotificationServiceConfig config, PreSharedKeyStore preSharedKeyStore) {
-        ChainedTokenInfo info = new ChainedTokenInfo(new PreSharedTokenInfoService(preSharedKeyStore),
-                                                     new OAuthTokenInfoService(config.getOauthInfoServiceUrl()));
+    TokenInfoService getTokenInfoService(final NotificationServiceConfig config, final PreSharedKeyStore preSharedKeyStore) {
+        final TokenInfoService info;
+        if(config.isDryRun()) {
+            info = new PreSharedTokenInfoService(new PreSharedKeyStore(
+                    ImmutableMap.of(DRY_RUN_PRE_SHARED_KEY, Long.MAX_VALUE)));
```

Comment:
I struggled a bit to understand how to make it work out of the box. I believe this, along with the note in the readme can help others to make it work out of the box. I only considered any potential risks for a bit. Do you think prod would ever use dry run?